### ISSUE TITLE
Add option to customize the default network and netmask

### ISSUE THREAD

**Issue by thiagomarafeli** [Angristan/OpenVPN-install#1201]
Nowadays, the script uses 10.8.0.0 network and 255.255.255.0 netmask for default, but it would be really nice if it had an option to customize this

**Comment by kondisz** [comment]
I had the same thing in my mind and yesterday I just used editor and replaced by search and replace all 10.8. By 10.88.
Installation went well and all works :)

**Comment by azfarhashmi** [comment]
I deployed the openvpn on one host and its working as expected, later I installed openvon on another host and that is also working fine but the problem is I can nit use both openvpns simultaneously and seems like as they are using same private network settings which is conflicting with other setup.

Is it possible the support custom private network during the setup so we can deploy multiple openvpn with unique private network?

**Comment by thiagomarafeli** [comment]
> I had the same thing in my mind and yesterday I just used editor and replaced by search and replace all 10.8. By 10.88. Installation went well and all works :)

@kondisz Yeah, I understand that's a "simple" thing to do, but, as there are other customized options (that may be used less than a custom IP range), I believe that would a nice improvement to the script

**Comment by thiagomarafeli** [comment]
> I deployed the openvpn on one host and its working as expected, later I installed openvon on another host and that is also working fine but the problem is I can nit use both openvpns simultaneously and seems like as they are using same private network settings which is conflicting with other setup.
> 
> Is it possible the support custom private network during the setup so we can deploy multiple openvpn with unique private network?

@azfarhashmi do you mean to have multiple openvpns in the same machine for multiples virtual networks? If so, I believe that would me more complex than this, as there may be many other things to consider in this use case

**Comment by angristan** [comment]
Done with https://github.com/angristan/openvpn-install/commit/e9deb4b8ab2bf1e7138da26fb2306010a904e853
